lib/tasks/install-blueprint.js

```diff
@@ -45,7 +46,16 @@ class InstallBlueprintTask extends Task {
 
     let blueprint = await this._resolveBlueprint(blueprintOption);
     logger.info(`Installing blueprint into "${target}" ...`);
-    return blueprint.install(installOptions);
+    let fileInfos = await blueprint.install(installOptions);
+
+    if (!options.skipLintFix) {
+      try {
+        let fileDisplayPaths = fileInfos.map(({ displayPath }) => displayPath);
+        await lintFix.run(fileDisplayPaths);
+      } catch (error) {
+        logger.error('Lint fix failed');
```

Comment:
Add a little more context RE: the error here

Reply:
done